package.json

```diff
@@ -27,11 +27,13 @@
     "webpack-dev-server": "2.9.7"
   },
   "engines": {
-    "yarn": "1.3.2"
+    "yarn": "^1.3.2"
   },
   "dependencies": {
     "@types/d3": "^5.0.0",
     "@types/d3-sankey": "^0.7.2",
-    "d3-sankey": "^0.7.1"
+    "d3-sankey": "^0.7.1",
+    "global": "^4.3.2",
+    "watch": "^1.0.2"
```

Comment:
I don't think `watch` needs to be added to the dependencies. A `watch` script that uses webpack's watch feature would be lighter weight:

`    "watch": "webpack --watch --progress --config webpack.config.js",`

Reply:
haha this is me doing `yarn add global watch` instead of `yarn global add watch`. will fix